test/unit/test_easy_logging.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+
+import os.path
+
+import pytest
+import tomlkit
+
+from snowflake.connector import EasyLoggingConfigPython
+from snowflake.connector.constants import CONFIG_FILE
+
+
+@pytest.fixture(scope="function")
+def setup1():
+    file_path = os.path.dirname(CONFIG_FILE)
+    if os.path.exists(CONFIG_FILE):
+        with open(CONFIG_FILE) as f:
+            connection_content = tomlkit.parse(f.read())
+            connection_content["log"] = {"log_level": "wrong"}
+        with open(CONFIG_FILE, "w") as f:
+            f.write(tomlkit.dumps(connection_content))
+    else:
+        os.makedirs(file_path, exist_ok=True)
+        with open(CONFIG_FILE, "w") as f:
+            connection_content = {
+                "log": {
+                    "level": "wrong",
+                }
+            }
+            f.write(tomlkit.dumps(connection_content))
+    yield
+    with open(CONFIG_FILE, "w") as f:
+        connection_content.pop("log")
+        f.write(tomlkit.dumps(connection_content))
+
+
+@pytest.fixture(scope="function")
+def setup2():
```

Comment:
Similarly, a more descriptive name would be helpful.  Also, is there any code you can share between the two fixtures, and can you think of a way to refactor them so as not to have to duplicate a bunch of code?